docs/README.md

```diff
@@ -156,26 +138,108 @@ forc index check
+|   ✅   | forc-postgres          |  /Users/rashad/.fuelup/bin/fuelup                       |
++--------+------------------------+---------------------------------------------------------+
+```
+
+### 2.2 Database setup
+
+To quickly setup and bootstrap the PostgreSQL database that we'll need, we'll use the `forc-postgres` plugin that is included in `fuelup`.
+
+> IMPORTANT: Ensure that any local PostgreSQL instance that is running on port `5432` is stoppe
+
+```bash
+forc index postgres create postgres --persistent
+```
+
+
+```text
+Downloading, unpacking, and bootstrapping database.
+▸▹▹▹▹ ⏱  Setting up database...                                                                                          🎬 Starting database at 'postgres://postgres:postgres@localhost:5432/postgres'.
+💡 Creating database at 'postgres://postgres:postgres@localhost:5432/postgres'.
+waiting for server to start....2023-02-09 16:11:25.764 EST [86849] LOG:  starting PostgreSQL 14.6 on x86_64-apple-darwin20.6.0, compiled by Apple clang version 12.0.0 (clang-1200.0.32.29), 64-bit
+2023-02-09 16:11:25.766 EST [86849] LOG:  listening on IPv6 address "::1", port 5432
+2023-02-09 16:11:25.766 EST [86849] LOG:  listening on IPv4 address "127.0.0.1", port 5432
+2023-02-09 16:11:25.766 EST [86849] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
+2023-02-09 16:11:25.769 EST [86850] LOG:  database system was shut down at 2023-02-09 16:06:37 EST
+2023-02-09 16:11:25.772 EST [86849] LOG:  database system is ready to accept connections
+ done
+server started
+▹▸▹▹▹ ⏱  Setting up database...                                                                                          2023-02-09 16:11:25.867 EST [86857] ERROR:  database "postgres" already exists
+2023-02-09 16:11:25.867 EST [86857] STATEMENT:  CREATE DATABASE "postgres"
+CREATE DATABASE "postgres"; rows affected: 0, rows returned: 0, elapsed: 368.775µs
+
+Default database postgres already exists.
+
+
+Writing PgEmbedConfig to "/Users/rashad/.fuel/idx/postgres/postgres-db.json"
+2023-02-09 16:11:25.868 EST [86857] LOG:  could not receive data from client: Connection reset by peer
+▪▪▪▪▪ ⏱  Setting up database...                                                                                          ✅ Successfully created database at 'postgres://postgres:postgres@localhost:5432/postgres'.
+2023-02-09 16:11:25.871 EST [86849] LOG:  received fast shutdown request
+2023-02-09 16:11:25.871 EST [86849] LOG:  aborting any active transactions
+2023-02-09 16:11:25.871 EST [86849] LOG:  background worker "logical replication launcher" (PID 86856) exited with exit code 1
+2023-02-09 16:11:25.872 EST [86851] LOG:  shutting down
+2023-02-09 16:11:25.877 EST [86849] LOG:  database system is shut down
+waiting for server to shut down.... done
+server stopped
+
+```
+
+Then we can start our database with
+
+```bash
+forc index postgres start postgres
+```
```

Comment:
when testing I ran all these commands with `forc-index`

Reply:
@rvmelkonian Yea when we build the plugin locally and copy it over the `fuelup`, we clobber fuleup and the commands are a bit different. But when this is released via fuelup - all commands are/will-be `forc index` (with space)